app/models/entry.server.ts

```diff
@@ -50,6 +50,29 @@ export function getEntries(skip?: number, take: number = 20) {
   })
 }
 
+export function getEntriesByInitialLettersAndPage(
+  initialLetters: string,
+  page: string
+) {
+  const pageNumber: number = parseInt(page);
```

Comment:
In **general**, you don't need to declare the type of every  variable you create — TypeScript is very good about inferring types, and usually the inferred type is fine. In this case,

```ts
const pageNumber = parseInt(page)
```

Explicit types do need to be declared for some things (e.g. function parameters), and are often helpful to declare for the return types of some functions — especially when you **know** exactly what you expect the function to return, and want to make sure your implementation of the function body doesn't accidentally return something else. For instance, when you want to ensure a function returns `string`, and not `string | undefined` because you forgot to handle a case where a sub-function call might return an undefined value.

Reply:
Done! Also good to know